### PR TITLE
About branch

### PR DESCRIPTION
**Pull request by meghz-glitch**
Check the code and need to add the photos. Please send photos in whatsapp

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -27,8 +28,79 @@
...
     <main>
-      <h2>Home</h2>
-    </main>
+
+      <section class="team-section">
+          <h1>Our Team</h1>
+          <div class="team-grid">
+              <!-- Team Member 1 -->
+              <div class="team-member">
+                  <img src="../assignment-gitguide/images/Megha.JPG" alt="May">
```
Comment (Vineethaveerapu) [inline about.html:37]: "Ensured that all image paths are relative to the project's internal structure (e.g., ./images/Megha.JPG) for better consistency and accessibility across the team."

Comment (Vineethaveerapu) [reply about.html:37]: @meghz-glitch

Comment (meghz-glitch) [reply about.html:37]: I will fix it but how will we add the pictures? Is it maually or I should do pull request for pictures too?

Comment (Vineethaveerapu) [reply about.html:37]: add pictures into the image folder then update the src of the images 
for example like this : `imagename` 
@meghz-glitch

**Pull request merged**